core/spec/models/spree/order_populator_spec.rb

```diff
@@ -18,5 +18,18 @@
         subject.populate(2, 5)
       end
     end
+
+    context 'with an invalid variant' do
+      let(:line_item) { build(:line_item) }
+
+      before do
+        allow(order.contents).to receive(:add).and_raise(ActiveRecord::RecordInvalid, line_item)
+      end
+
+      it 'has some errors' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.